app/models/hearing.js

```diff
@@ -0,0 +1,20 @@
+import DS from 'ember-data';
+import { attr, belongsTo } from '@ember-decorators/data';
+
+const { Model } = DS;
+
+export default class HearingModel extends Model {
+// DB Table: dcp_communityboarddisposition
+
+  // One Action to One Hearing
+  @belongsTo('action') action;
+
+  // calculated based on date
+  @attr('boolean') isScheduled;
```

Comment:
Where will the calculation happen? In the form controller? I thought it could be done here in the model but maybe not.

Reply:
In the spreadsheet, it mentioned it was "calculted based on date"--I'm assuming this means the public hearing date?